src/cuga/backend/server/auth/dependencies.py

```
@@ -0,0 +1,93 @@
+security = HTTPBearer(auto_error=False)
+
+_validator_cache: dict[str, JWTValidator] = {}
+
+
+def _auth_enabled() -> bool:
+    try:
+        from cuga.config import settings
+
+        auth = getattr(settings, "auth", None)
+        return bool(auth and getattr(auth, "enabled", False))
+    except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)